### DISCUSSION TITLE
Define rules brainstorm for across study checks

### DISCUSSION THREAD

**Discussion by rbakercdisc1** [cdisc-org/ddf-core-poc#101] [Ideas]
There may be pieces for types of study that would require a check across the USDM for an across study check. Here are some ideas from the brainstorming session.

1.If it is a cross over study there should be two interventions.
2. If a study element is associated with a study intervention, then the start of the element should be the start of the intervention.
3. Different trial types versus the existence of an indication. 
4. An example, if the trial type is treatment, there should be an indication of what is being treated.  
5. Trial intent types - basic science, cure, diagnosis, treatment, mitigation, Trials of this with this intent type would be expected to have an indication. The "indication" should be "indicated". At least one instance of the "indication" class should be connected to study design.
6. Check values of trial summary parameters for consistency.
7. It is common for a study cell to have a study element, if it has more than one study element, the relationship between the study elements are the study start and end rules. What are the constraints? What are the transition rules? The endrule could say, "after this period of time."
8. If  the study cells have to be populated, what is in the study cell and how do those relate?

@ASL-rmarshall 
@DianeWold 
Please add to/edit this discussion. Thanks.

**Comment by rbakercdisc1** [comment]
Added PBI to highlight the discussion  - Define rules for across specific study type USDM checks#102

**Comment by DianeWold** [comment]
Many of these are checks that a trial summary parameter matches the study design.

1. For a StudyDesign, if TSPARM = "Intervention Model" and TSVAL = "CROSSOVER" then there must be at least two StudyInterventions.
2. If a StudyElement has a relationship to one or more StudyInterventions, then the TransitionStartRule for the StudyElement must be that the element starts with the start of one of its associated StudyInterventions.
3. For a StudyDesign, if TSPARM = "Trial Type" and TSVAL = "EFFICACY" or "TREATMENT" or other relevant values, then TSPARM = "Trial Disease/Condition Indication" must be present and populated.
4. similar to 3
5.  For a StudyDesign, if TSPARM = "Trial Intent Type" and TSVAL ="TREATMENT" or "PREVENTION" or other relevant values,  then TSPARM = "Trial Disease/Condition Indication" must be present and populated.
6. general statement of which 1, 3, and 5 are examples
7. musings on how multi-element study cells are ordered.  Presumable the transition rules handle this.
8.  following on from the existing rule that for each arm of a study design, there must be a study cell for each epoch. Similar to 7.

**Comment by BSnoeijerCD** [comment]
@DianeWold : see draft IG part for mapping to SDTM: https://wiki.cdisc.org/display/USDMIGv3/Creation+of+SDTM+Trial+Design+Domains

**Comment by BSnoeijerCD** [comment]
@DianeWold @rbakercdisc1 @ASL-rmarshall 
1.
Intervention model is stored in the attribute studyDesign.interventionModel. The corresponding allowed terminology can be found here: https://ncit.nci.nih.gov/ncitbrowser/ajax?action=create_src_vs_tree&vsd_uri=http://evs.nci.nih.gov/valueset/CDISC/C99076

For this I would suggest a rule like: 
-	If StudyDesign.interventionModel is “FACTORIAL”, “CROSS-OVER”, “PARALLEL”or “SEQUENTIAL” then at least 2 StudyInterventions should be specified
2.
Not sure if we want to restrict this. Could be that pre-dose assessments are included in the epoch/element. So might be tricky.
3-5.
the CT list is quite long and more than 1 can be specified. Not sure whether TREATMENT requires an indication. for Phase I treatment can also be given to healthy volunteers. EFFICACY is more towards that direction.
